frontend/javascripts/oxalis/view/action-bar/toolbar_view.tsx

```diff
@@ -504,6 +511,14 @@ function ChangeBrushSizeButton() {
 export default function ToolbarView() {
   const hasVolume = useSelector((state: OxalisState) => state.tracing.volumes.length > 0);
   const hasSkeleton = useSelector((state: OxalisState) => state.tracing.skeleton != null);
+  const hasAgglomerateMappings = useSelector((state: OxalisState) => {
+    const visibleSegmentationLayer = getVisibleSegmentationLayer(state);
+    return (
+      visibleSegmentationLayer != null &&
+      visibleSegmentationLayer.agglomerates != null &&
+      visibleSegmentationLayer.agglomerates.length > 0
```

Comment:
```suggestion
      visibleSegmentationLayer?.agglomerates?.length > 0
```

I hope I'm not making any false claims with my suggestions, but I'm starting to dig the `?.` operator :smile:

Reply:
Indeed, very helpful! I'll have to remember to use it :)